feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/section/TimetableHeader.kt

```diff
@@ -31,10 +43,30 @@ fun TimetableHeader(modifier: Modifier = Modifier) {
                 style = MaterialTheme.typography.labelMedium,
             )
         }
-        Image(
-            modifier = Modifier.size(width = 185.dp, height = 169.dp),
-            painter = painterResource(id = R.drawable.img_droid_kun_in_bath),
-            contentDescription = null,
-        )
+        Box {
+            var atEnd by remember { mutableStateOf(false) }
+            Image(
+                modifier = Modifier.size(width = 185.dp, height = 169.dp),
+                painter = painterResource(id = R.drawable.img_droid_kun_in_bath),
```

Comment:
Thanks for amazing animation. How about using png or webp for img_droid_kun_in_bath to improve performance? I'm not sure but the warning will occur if the vector drawable is big.